code/Equipment/Weapons/WeaponComponent.cs

```diff
@@ -153,12 +166,12 @@ public Vector3 GetStartPosition( bool isDroppable = false )
 		if ( Equipment.Grub is not { } grub )
 			return Vector3.Zero;
 
-		var muzzle = Equipment.Model.GetAttachment( "muzzle" );
+		Transform? muzzle = Equipment.Model.GetAttachment( "muzzle" );
```

Comment:
Use var when possible

Reply:
I'm annoyed. My code cleanup does this for some reason (I have the editorconfig from the github) and must have clicked it by mistake. Fixed in https://github.com/apetavern/grubs/pull/355/commits/23dbbb2ef5dd9dddee2079e4c739e9fe0dcb0a3c